Subject: [MAKEFLOPPIES] Remove XSIisms and replace use of /bin/bash with /bin/sh
Author: David Weinehall <tao@acc.umu.se>
Bug-Debian: http://bugs.debian.org/254933
---
diff --git a/src/MAKEFLOPPIES b/src/MAKEFLOPPIES
index ca1e557..9127d97 100755
--- a/src/MAKEFLOPPIES
+++ b/src/MAKEFLOPPIES
@@ -54,7 +54,7 @@ getword ()
 {	if [ $# -lt 1 ]; then
 		return
 	fi
-	if [ "$1" -lt 1 -o "$1" -ge $# ]; then
+	if [ "$1" -lt 1 ] || [ "$1" -ge $# ]; then
 		return
 	fi
 	shift $1
@@ -108,7 +108,7 @@ cmosid ()
 # main()
 
 PERMISSION=666
-while [ $# -ge 1 -o -n "${REMAINDER}" ]; do
+while [ $# -ge 1 ] || [ -n "${REMAINDER}" ]; do
 	if [ -n "$REMAINDER" ]; then
 		## Continue processing options stuck together
 		ARG=$REMAINDER
@@ -210,13 +210,13 @@ for DRIVE in $DRIVES; do
 	fi
 
 	FORMATS=`getword "$CMOS" $CMOSFORMATS`
-	if [ -n "$VERBOSE" -o -n "$DRYRUN" ]; then
+	if [ -n "$VERBOSE" ] || [ -n "$DRYRUN" ]; then
 		echo rm -f "$FILE"*
 	fi
 	if [ -z "$DRYRUN" ]; then
 		rm -f "$FILE"*
 	fi
-	if [ -z "$CMOS" -o -z "$CN" ]; then
+	if [ -z "$CMOS" ] || [ -z "$CN" ]; then
 		echo "Skipping invalid drive \"$FILE\"" >&2
 		continue
 	fi
@@ -226,7 +226,7 @@ for DRIVE in $DRIVES; do
 		chown root:floppy "$FILE"
 		chmod ${PERMISSION} "$FILE"
 	fi
-	if [ -n "$VERBOSE" -o -n "$DRYRUN" ]; then
+	if [ -n "$VERBOSE" ] || [ -n "$DRYRUN" ]; then
 		echo mknod "$FILE" b "$MAJOR" "$BASENUMBER"
 	fi
 
@@ -250,7 +250,7 @@ for DRIVE in $DRIVES; do
 				chown root:floppy "${FILE}${NAME}"
 				chmod ${PERMISSION} "${FILE}${NAME}"
 			fi
-			if [ -n "$VERBOSE" -o -n "$DRYRUN" ]; then
+			if [ -n "$VERBOSE" ] || [ -n "$DRYRUN" ]; then
 				echo mknod "${FILE}${NAME}" b "$MAJOR" "$MINOR"
 			fi
 		fi
